### ISSUE TITLE
When the file has bad track (can not read data completely)

### ISSUE THREAD

**Issue by adan1** [OV2/RapidCRC-Unicode#62]
RapidCRC stop calculating long long time and can not exit

**Comment by vatterspun** [comment]
What's the medium?  Is this a scratched CD/DVD?

**Comment by vatterspun** [comment]
> RapidCRC stop calculating long long time and can not exit

What's the medium? Is this a scratched CD/DVD?

As a workaround, you might try a program that ignores copy errors such as Roadkil's Unstoppable Copier (http://roadkil.net/program.php?ProgramID=29) and then run the RapidCRC analysis.  That program is a few years old now and there may be a better solution, but it's where I'd start.

**Comment by adan1** [comment]
> > RapidCRC stop calculating long long time and can not exit
> 
> What's the medium? Is this a scratched CD/DVD?
> 
> As a workaround, you might try a program that ignores copy errors such as Roadkil's Unstoppable Copier (http://roadkil.net/program.php?ProgramID=29) and then run the RapidCRC analysis. That program is a few years old now and there may be a better solution, but it's where I'd start.

It's HDD. I know some data retrieving software to help. But i want to know which files are bad already at once, so why not just let RapidCRC stop waiting for read data if file has bad track, skip this file, show error message and verify the remaining files?

**Comment by vatterspun** [comment]
Yeah the program freezing during operation is not good.  I'm not the dev but I think I can help a little with this process:

Few questions:

1. What is your setup?  Operating system?  What version of the program are you testing?  Can you share a screenshot?
2. Is it possible to post one of the corrupted files so the dev can try to recreate your experience?

**Comment by adan1** [comment]
> Few questions:
> 1. What is your setup?  Operating system?  What version of the program are you testing?  Can you share a screenshot?
> 2. Is it possible to post one of the corrupted files so the dev can try to recreate your experience?


It's Unicode 0.3.27 version. The bad file can not be read at some position, just stuck there, because physical damaged, so i can not send this file. I also post this problem at MultiPar forum, its log return 0x17 error code, now it had solved, just waiting for the new version.

A part of report by AIDA64:
    Computer:
      Computer Type                                     ACPI x64-based PC
      Operating System                                  Microsoft Windows 7 Ultimate
      OS Service Pack                                   Service Pack 1
      Internet Explorer                                 11.0.9600.19204
      DirectX                                           DirectX 11.1
    Motherboard:
      CPU Type                                          QuadCore Intel Core i5-4590, 3500 MHz (35 x 100)
      Motherboard Name                                  Gigabyte GA-Z97-D3H  (3 PCI, 2 PCI-E x1, 2 PCI-E x16, 1 M.2, 4 DDR3 DIMM, Audio, Video, Gigabit LAN)
      Motherboard Chipset                               Intel Wildcat Point Z97, Intel Haswell
      System Memory                                     16244 MB  (DDR3-1600 DDR3 SDRAM)
      DIMM2: Kingston 99U5471-058.A00LF                 8 GB DDR3-1600 DDR3 SDRAM  (11-11-11-28 @ 800 MHz)  (10-10-10-27 @ 761 MHz)  (9-9-9-24 @ 685 MHz)  (8-8-8-22 @ 609 MHz)  (7-7-7-19 @ 533 MHz)  (6-6-6-16 @ 457 MHz)
      DIMM4: Kingston 99P5471-033.A00LF                 8 GB DDR3-1600 DDR3 SDRAM  (11-11-11-28 @ 800 MHz)  (10-10-10-27 @ 761 MHz)  (9-9-9-24 @ 685 MHz)  (8-8-8-22 @ 609 MHz)  (7-7-7-19 @ 533 MHz)  (6-6-6-16 @ 457 MHz)
    Display:
      Video Adapter                                     Intel(R) HD Graphics 4600  (1 GB)
      Video Adapter                                     NVIDIA GeForce GTX 760  (2 GB)
    Storage:
      IDE Controller                                    Intel(R) 9 Series Chipset Family SATA AHCI Controller
      Disk Drive                                        PLEXTOR PX-128M6S SCSI Disk Device  (128 GB, SATA-III)
      Disk Drive                                        ST2000DM 006-2DM164 SCSI Disk Device  (1863 GB)
      Disk Drive                                        WDC WD10EZEX-08M2NA0 SCSI Disk Device  (1000 GB, 7200 RPM, SATA-III)
      Disk Drive                                        WDC WD20EZRZ-00Z5HB0 SCSI Disk Device  (1863 GB)
      SMART Hard Disks Status                           OK

**Comment by vatterspun** [comment]
It's _physically_ damaged and yet SMART Hard Disk Status is "OK"?  That's strange.  If it's physical drive damage, that might be hard for the dev to replicate and resolve.

Also, can you link to the Multipar thread?  Not that I think that's going to provide some special insight, I'm just curious.

**Comment by adan1** [comment]
[https://www.livebusinesschat.com/smf/index.php?topic=6368](url)

**Comment by vatterspun** [comment]
> and the post https://www.livebusinesschat.com/smf/index.php?topic=6368

Got it, thanks
